README.md

```diff
@@ -139,38 +139,57 @@ await waitForActions(store, [
 ]);
 ```
 
-### Example #3: function (advanced use cases)
+### Example #3: custom action matcher (advanced use cases)
 
-Supply a predicate which will be called for every action dispatched.
+Supply a matcher function which will be called for every action dispatched. The function accepts two arguments: the array of expected actions and dispatched actions.
 
 ```js
 import waitForActions from 'redux-mock-store-await-actions';
 import configureStore from 'redux-mock-store';
 import thunkMiddleware from 'redux-thunk';
 
 const store = configureStore([thunkMiddleware])();
+const expectedActions = [
+    { type: 'LOGIN_START', payload: { username, password } },
+    { type: 'FETCH_ORDERS_SUCCESS' }
+];
 
-store.dispatch(login('my-username', '123'));
-await waitForActions(store, (storeActions) => {
-	const hasLoginStart = storeActions.some((item) => item.type === 'LOGIN_START' && item.payload.username === 'my-username');
-    const hasFetchOrdersSuccess = storeActions.some((item) => item.type === 'FETCH_ORDERS_SUCCESS');
+store.dispatch(login('my-username', 'my-password'));
+// Matches and LOGIN_START and FETCH_ORDERS_SUCCESS are dispatched
+await waitForActions(store, expectedActions, (expectedActions, storeActions) => {
```

Comment:
Isn't the third argument, an options object with a `matcher`?

Reply:
Yep. Will fix.